lib/utils/logDeprecation.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const chalk = require('chalk');
+const memoizee = require('memoizee');
+
+const loggedDeprecations = {};
```

Comment:
It'll be safer to use _set_ (using plain object as above, is immune to issues, e.g. `loggedDeprecation.hasOwnProperty` will always be truthy)

Reply:
Done